pkg/klusterlet/clusterclaim/clusterclaimer.go

```diff
@@ -51,6 +51,11 @@ const (
 	labelExcludeBackup = "velero.io/exclude-from-backup"
 )
 
+// consistList returns a list of cluster claims that only need to create, not update.
+var consistList = []string{
+	ClaimK8sID,
```

Comment:
what does consistList mean? clusterClaimCreateOnlyList?

Reply:
Yes, clusterClaimCreateOnlyList is better.